### PR TITLE
adds rake task to clear unclaimed listings

### PR DESCRIPTION
**Pull request by nhiquach**
### What does this PR do?
Adds a rake task (to be run every day at 12pm to take unclaimed listings and reassign to replate fleet (DoorDash Maen's driver id)

### Status
NEEDS TESTING

### Migrations
NO

### Todos
- [ ] Testing
- [ ] Documentation/`annotate`

CC: @nhiquach

### CODE REVIEW

File: lib/tasks/daily_marketplace_task.rake [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  task clear_listings: :environment do
+    # This task is called at 8:00PM UTC (12:00PM PST) every day
+
+    unclaimed = MarketplaceListing.all.select { |listing| \
+                listing.complete_before.to_date == Time.now.utc.to_date && \
+                listing.state == 'open'}
+    replate_onfleet_driver_id = 'k95yhp31QHgu9yTjz8Nswi~H'
+    unclaimed.each do |listing|
+      listing.marketplace_region_id = nil
+      MarketplaceListing.onfleet_update_listing(listing, replate_onfleet_driver_id)
```
Comment (nhiquach) [inline lib/tasks/daily_marketplace_task.rake]: relies on changes from michelle's branch for cancelClaim

Comment (nhiquach) [reply lib/tasks/daily_marketplace_task.rake]: resolved - changed merged to master

File: lib/tasks/daily_marketplace_task.rake [hunk 2]
```diff
@@ -0,0 +1,15 @@
+namespace :daily_marketplace_task do
+  desc 'Moves listings unclaimed by noon on day of pickup to Replate fleet'
+  task clear_listings: :environment do
+    # This task is called at 8:00PM UTC (12:00PM PST) every day
+
+    unclaimed = MarketplaceListing.all.select { |listing| \
+                listing.complete_before.to_date == Time.now.utc.to_date && \
+                listing.state == 'open'}
+    replate_onfleet_driver_id = 'k95yhp31QHgu9yTjz8Nswi~H'
```
Comment (nhiquach) [inline lib/tasks/daily_marketplace_task.rake]: change to ENV variable?

**Comment by michaeldo2** [comment]
closing because not done yet

**Pull request closed without merging**